docs/sandbox-permissions.rst

```diff
@@ -230,6 +230,7 @@ You can provide the following device permissions:
 ``kvm``   Kernel based Virtual Machine ``/dev/kvm``
 ``shm``   Shared Memory in ``/dev/shm``.
 ``input`` Input devices as exposed in ``/dev/input``. This includes game controllers. Since Flatpak 1.15.6.
+``usb``   Raw USB devices as exposed in ``/dev/bus/usb``. Since Flatpak 1.15.8.
```

Comment:
Maybe link to [release notes](https://github.com/flatpak/flatpak/releases/tag/1.15.8)?

Reply:
no because it's not in 1.15.8. This PR is still draft as the feature hasn't landed.